web/src/admin/EditCustomer.vue

```diff
@@ -64,29 +87,88 @@ export default {
         this.formErrors.push("Name required")
       }
     },
+    toggleService(id, event) {
+      let newState = event.target.checked
+      console.log("toggle", id, event.target.checked)
+      let service = this.availableServices.find(v => v.id == id)
+      if (!service) {
+        console.warn("service not found")
```

Comment:
doesn't need UI feedback?

Reply:
Very unlikely, but removed the console error.